routes/gameRoutes.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var express = require('express');
+var gameMatch = require ('../node_modules/colorGen/color');
```

Comment:
If this is an npm installed module, you don't provide a path to the module. Instead it's just:

`var gameMatch = require('color');`

or whatever the name of the installed module is. Also, if it's an installed module, it has to appear in the dependencies list of your package.json file, or it won't install for me when I run npm install. If it's not an npm installed module, then it shouldn't be in node_modules.